Older reporting configurations were not properly handling username

zomglings opened the pull request:
## 🚀 🚀 Pull Request

This is a potential fix to this issue. It is based on this PR: https://github.com/activeloopai/Hub/pull/1800

### Checklist:

- [ ]  [My code follows the style guidelines of this project](https://www.notion.so/activeloop/Engineering-Guidelines-d6e502306d0e4133a8ca507516d1baab) and the [Contributing document](https://github.com/activeloopai/Hub/blob/release/2.0/CONTRIBUTING.md)
- [ ]  I have commented my code, particularly in hard-to-understand areas
- [ ]  I have kept the `coverage-rate` up
- [ ]  I have performed a self-review of my own code and resolved any problems
- [ ]  I have checked to ensure there aren't any other open [Pull Requests](https://github.com/activeloopai/Hub/pulls) for the same change
- [ ]  I have described and made corresponding changes to the relevant documentation
- [ ]  New and existing unit tests pass locally with my changes


### Changes

<!-- Describe your changes! Describe the scope of this PR's responsibilities. -->

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/activeloopai/Hub?pullRequest=1806) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/activeloopai/Hub?pullRequest=1806) before we can accept your contribution.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: adolkhan<br/>:x: zomglings<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/activeloopai/Hub?pullRequest=1806) it.</sub>

zomglings commented:
I updated this PR so that the code also:
1. Associates the original client ID as a machine ID.
2. No longer overwrites the client ID with the username in the reporting config file.

The second change ensures that config files for new users have the same structure as config files for old users (who started using `activeloop` before these changes make it into `main`).

For convenience, here is my commit message on the most recent change:

     Adds "machine_id:<original client id>" tag to reports

    I also reverted the change where client_id is overwritten by username
    when the config file is written, as we need to retain a distinct
    machine_id for each machine the user uses `activeloop` on.

    This remapping of keys inside reporting_config should probably be
    refactored at some point because it could be confusing for future
    builders on this project. For now, it was the simplest way to get the
    desired behavior in the analytics data. The more significant (and
    correct) change will require more testing than this.

tatevikh commented:
@zomglings thanks for the contribution. Can you pls sign the CLA, so we can successfully merge the PR. Thanks!
